pkg/render/fluentd.go

```diff
@@ -70,6 +70,11 @@ const (
 	SplunkFluentdSecretsVolName              = "splunk-certificates"
 	SplunkFluentdDefaultCertDir              = "/etc/ssl/splunk/"
 	SplunkFluentdDefaultCertPath             = SplunkFluentdDefaultCertDir + SplunkFluentdSecretCertificateKey
+	SysLogInternetCADir                      = "/etc/pki/tls/certs/"
+	SysLogInternetCertKey                    = "ca-bundle.crt"
+	SysLogInternetCAPath                     = SysLogInternetCADir + SysLogInternetCertKey
+	SyslogCAConfigMapName                    = "syslog-ca"
+	SyslogCABundleName                       = "tls.crt"
```

Comment:
We can delete this constant and use the one from the corev1 package: TLSCertKey

Reply:
Done